N4.md

```diff
@@ -0,0 +1,152 @@
+# Chapter 3
+## Canvas
+HTML 5 defines the <canvas> element as “a resolution-dependent bitmap canvas which can be used for rendering graphs, game graphics, or other visual images on the fly, So what does a canvas look like? Nothing, really. A <canvas> element has no content and no border of its own.
+```html
+<canvas width="300" height="225"></canvas>
+```
+to draw something in the canvas you can use a code like this.
+```html
+function draw_b() {
+  var b_canvas = document.getElementById("b");
+  var b_context = b_canvas.getContext("2d");
+  b_context.fillRect(50, 25, 150, 100);
+}
+```
+Donde se dibuja un cuadrado negro de esas dimensiones dentro del canvas.
+
+So, you have a <canvas> element, and you have its drawing context. The drawing context is where all the drawing methods and properties are defined. There’s a whole group of properties and methods devoted to drawing rectangles:
+
+- The fillStyle property can be a CSS color, a pattern, or a gradient. (More on gradients shortly.) The default fillStyle is solid black, but you can set it to whatever you like. Each drawing context remembers its own properties as long as the page is open, unless you do something to reset it.
+- fillRect(x, y, width, height) draws a rectangle filled with the current fill style.
+- The strokeStyle property is like fillStyle — it can be a CSS color, a pattern, or a gradient.
+- strokeRect(x, y, width, height) draws an rectangle with the current stroke style. strokeRect doesn’t fill in the middle; it just draws the edges.
+- clearRect(x, y, width, height) clears the pixels in the specified rectangle.
+
+## Canvas Coordinates
+The canvas is a two-dimensional grid. The coordinate (0, 0) is at the upper-left corner of the canvas. Along the X-axis, values increase towards the right edge of the canvas. Along the Y-axis, values increase towards the bottom edge of the canvas.
+## Paths

```

Comment:
Estaría bien que agregaras un folder de assets y dentro del folder un subfolder por cada capítulo que requiera media, por ejemplo:

assets/chapter-4/paths.jpg

y que [agregaras una imagen](https://stackoverflow.com/questions/14494747/how-to-add-images-to-readme-md-on-github) a las partes de este capítulo que generan formas para que se queden como referencia.